packages/core/src/extensions/Blocks/nodes/BlockContent/defaultRenderHTML.ts

```diff
@@ -0,0 +1,40 @@
+import { mergeCSSClasses } from "../../../../shared/utils";
+
+export function defaultRenderHTML(
```

Comment:
let's stay focused on naming. "default" doesn't say a lot, and "renderHTML" is very much focused on where it's used (i.e., on the "outside"), instead of explaining what the function itself is. Can you come up with a better name?